build/pkgs/ecm/spkg-install.in

```diff
@@ -92,10 +92,10 @@ else
             # Don't add debug symbols because configure otherwise
             # fails due to a bus error in Apple's 'ld' when trying
             # to determine if global symbols are prefixed with an
-            # underscore (cf. http://trac.sagemath.org/sage_trac/ticket/5847#comment:35 ff.):
+            # underscore (cf. https://github.com/sagemath/sage/issues/5847 ff.):
```

Comment:
You can keep the fragments in the URL. We made sure that they also work.

Reply:
Oh great!  I'll go back and undo those changes.